requests_oidc/factory.py

```diff
@@ -19,18 +19,23 @@ def make_client_credentials_session(
     client_id: str,
     client_secret: str,
     updater: TokenUpdater | None = None,
+    scope: list[str] | None = None,
+    **kwargs,
 ) -> OAuth2Session:
+    scope_ = ["openid"] if scope is None else scope + ["openid"]
```

Comment:
I don't know if double-specifying scope is safe, I'll take it from here & get this in